config/locales/server.en.yml

```diff
@@ -3,3 +3,4 @@ en:
     discourse_global_filter_enabled: "Enable the Global Filter plugin"
     global_filters: "The tag(s) that will be added to the global-filter-container"
     display_subcategories_for_tag_intersection: "Display a list of a category's subcategories on category/tag intersection routes"
+    replace_global_filter_with_children: "When building a list of available global filters in the composer, replace a global filter with its filter children if present"
```

Comment:
Provide the option to add filter children but not hide the parent and vice versa